### PR TITLE
fix #3116

### LINKED ISSUE #3116: Export light-my-request types
## 🚀 Feature Proposal

Export light-my-request types

## Motivation

When I write fastify tests with `.inject` in some circumstances I need access to the `InjectOptions` type. Currently, I need to install the package separately.

### PR DESCRIPTION
**Pull request by StarpTech**
#### Checklist

- [X] run `npm run test` and `npm run benchmark`
- [X] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [X] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: fastify.d.ts [hunk 1]
```diff
@@ -159,6 +158,7 @@ export interface ValidationResult {
 }
 
 /* Export all additional types */
+export { Chain as LightMyRequestChain, InjectOptions, Response as LightMyRequestResponse, CallbackFunc as LightMyRequestCallback } from 'light-my-request'
```
Comment (StarpTech) [inline fastify.d.ts:161]: Export same as import in https://github.com/fastify/fastify/blob/main/types/instance.d.ts#L1

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by mcollina** [comment]
cc @fastify/typescript for a check

File: fastify.d.ts (hunk 1 above)
Comment (climba03003) [reply fastify.d.ts:161]: I would suggest using the syntax below as we only re-export the type.
```
export type { .. } from 'light-my-request'
```

Comment (StarpTech) [reply fastify.d.ts:161]: Good point

**Review by zekth — APPROVED** [review]
lgtm

**Review by L2jLiga — APPROVED** [review]
LGTM, ~however I would like to merge it to the next cause it might be breaking change for typescript users~ sorry, was wrong

**Review by Ethan-Arrowood — APPROVED** [review]

**Pull request merged**